### ISSUE TITLE
@storybook/core – dll/storybook_ui-manifest.json does not exist after installation

### ISSUE THREAD

**Issue by sawyerh** [storybookjs/storybook#9763]
**Describe the bug**
After bootstrapping a project with `@storybook/react` `v5.3.11` and running `start-storybook -p 6006`, I received the following error in the CLI:

```
TypeError: Cannot read property 'toJson' of null
    at Watching.handler (/[my project]/node_modules/@storybook/core/dist/server/dev-server.js:84:17)
```

After adding additional logging to `dev-server.js` to get some more useful debug info, it appears that this is due to the following error:

```
Error: ENOENT: no such file or directory, open '/[my project]/node_modules/@storybook/core/dll/storybook_ui-manifest.json'
```

When I look in the `@storybook/core/dll/` directory, there is nothing in it. 

I've tried deleting my `node_modules` and installing from scratch, but get the same results.

**System:**

```
  System:
    OS: macOS 10.15.2
    CPU: (4) x64 Intel(R) Core(TM) i5-6267U CPU @ 2.90GHz
  Binaries:
    Node: 12.13.0 - ~/.nvm/versions/node/v12.13.0/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.13.7 - ~/.nvm/versions/node/v12.13.0/bin/npm
  Browsers:
    Chrome: 80.0.3987.87
    Firefox: 71.0
    Safari: 13.0.4
  npmPackages:
    @storybook/react: ^5.3.11 => 5.3.11
```

**Additional context**
Potentially related to #9707?

**Comment by sawyerh** [comment]
Looks like this was fixed in `5.3.12`

**Comment by shilman** [comment]
Definitely related to #9707 - I need to be super careful with the release until this gets fixed. cc @ndelangen
